packages/core/src/domain/tracekit/tracekit.ts

```diff
@@ -1,96 +0,0 @@
-import { instrumentMethodAndCallOriginal } from '../../tools/instrumentMethod'
-import { computeStackTrace } from './computeStackTrace'
-import type { UnhandledErrorCallback, StackTrace } from './types'
-
-// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error#Error_types
-const ERROR_TYPES_RE =
-  /^(?:[Uu]ncaught (?:exception: )?)?(?:((?:Eval|Internal|Range|Reference|Syntax|Type|URI|)Error): )?([\s\S]*)$/
-
-/**
- * Cross-browser collection of unhandled errors
- *
- * Supports:
- * - Firefox: full stack trace with line numbers, plus column number
- * on top frame; column number is not guaranteed
- * - Opera: full stack trace with line and column numbers
- * - Chrome: full stack trace with line and column numbers
- * - Safari: line and column number for the top frame only; some frames
- * may be missing, and column number is not guaranteed
- * - IE: line and column number for the top frame only; some frames
- * may be missing, and column number is not guaranteed
- *
- * In theory, TraceKit should work on all of the following versions:
- * - IE5.5+ (only 8.0 tested)
- * - Firefox 0.9+ (only 3.5+ tested)
- * - Opera 7+ (only 10.50 tested; versions 9 and earlier may require
- * Exceptions Have Stacktrace to be enabled in opera:config)
- * - Safari 3+ (only 4+ tested)
- * - Chrome 1+ (only 5+ tested)
- * - Konqueror 3.5+ (untested)
- *
- * Tries to catch all unhandled errors and report them to the
- * callback.
- *
- * Callbacks receive a StackTrace object as described in the
- * computeStackTrace docs.
- *
- * @memberof TraceKit
- * @namespace
```

Comment:
This comment is no longer accurate since https://github.com/DataDog/browser-sdk/pull/1335/files#diff-c33db8fef9eab4ff4982984c1dd39c92172aaefd0f88d528f205dcffa82d35cd. Let's remove it